app.js

```diff
@@ -45,17 +47,22 @@ if (program.address) {
 }
 
 if (program.peers) {
-	if (typeof program.peers === 'string') {
-		appConfig.peers.list = program.peers.split(',').map(function (peer) {
-			peer = peer.split(':');
+	appConfig.peers.list = parametersReader.convertToAddressList(program.peers, appConfig.port);
+}
+
+if (program.sync) {
+	appConfig.syncPeers = {
+		list: parametersReader.convertToAddressList(program.sync, appConfig.port).map(function (syncPeer) {
```

Comment:
Why not doing it inside `parametersReader` if we already have that for this specified case.